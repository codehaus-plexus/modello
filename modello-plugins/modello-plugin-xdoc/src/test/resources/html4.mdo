<?xml version="1.0"?>

<model xmlns="http://codehaus-plexus.github.io/MODELLO/1.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://codehaus-plexus.github.io/MODELLO/1.0.0 http://codehaus-plexus.github.io/modello/xsd/modello-1.0.0.xsd">
  <id>html4</id>
  <name>Modello Java Mojo Test Model</name>
  <root>Model</root>
  <defaults>
    <default>
      <key>package</key>
      <value>org.codehaus.mojo.modello.javatest</value>
    </default>
  </defaults>
  <classes>
    <class rootElement="true">
      <comment>This is a comment</comment>
      <description>
        <![CDATA[
            Whether this proxy configuration is the active one. Note: While the type of this field
            is {@code String} for technical reasons, the semantic type is actually {@code boolean}.<br>
            This is a description in HTML4<br>
            <strong>NOTE: </strong> HTML linebreak should be converted to selfclosing XML-tag
            @since Maven 3
          ]]>
      </description>
      <name>Model</name>
      <version>0.9.0+</version>
      <fields>
        <field>
          <name>extend</name>
          <version>1.0.0+</version>
          <type>String</type>
        </field>
      </fields>
    </class>
    <class>
      <name>NewModel</name>
      <version>1.0.0+</version>
    </class>
  </classes>
</model>
