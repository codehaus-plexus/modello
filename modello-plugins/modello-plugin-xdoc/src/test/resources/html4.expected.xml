<?xml version="1.0"?>
<!-- =================== DO NOT EDIT THIS FILE ====================         -->
<!-- Generated by Modello,                                                  -->
<!-- any modifications will be overwritten.                                 -->
<!-- ==============================================================         -->
<document>
  <properties>
    <title>Modello Java Mojo Test Model</title>
  </properties>
  <body>
    <section name="Modello Java Mojo Test Model">
      <p>No description.</p>
      <source>
&lt;<a href="#class_model">model</a>&gt;
  &lt;extend/&gt;
&lt;/model&gt;
</source>
      <a name="class_model"/>
      <subsection name="model">
        <p>Whether this proxy configuration is the active one. Note: While the type of this field is <code>String</code> for technical reasons, the semantic type is actually <code>boolean</code>.
<br />
This is a description in HTML4
<br />
<strong>NOTE: </strong> HTML linebreak should be converted to selfclosing XML-tag
<br />
<p><b>Since</b>: Maven 3</p></p>
        <table>
          <tr>
            <th>Element</th>
            <th>Type</th>
            <th>Since</th>
            <th>Description</th>
          </tr>
          <tr>
            <td>
              <code>extend</code>
            </td>
            <td>
              <code>String</code>
            </td>
            <td>1.0.0</td>
            <td>No description.</td>
          </tr>
        </table>
      </subsection>
    </section>
  </body>
</document>
